lib/matplotlib/tests/test_tickedstroke.py

```diff
@@ -0,0 +1,64 @@
+from matplotlib.testing.decorators import image_comparison
+import matplotlib.pyplot as plt
+import matplotlib.patheffects as PathEffects
+from matplotlib.path import Path
+import matplotlib.patches as patches
+import numpy as np
+
+
+@image_comparison(['tickedstroke'], remove_text=True)
```

Comment:
Consider only making this a test on png?  

I think this test should still be in the path effects.

Reply:
Done in f765ae01 below.

Thanks